remo/profiles/tests/test_models.py

```diff
@@ -281,19 +281,23 @@ def setUp(self):
         """Setup tests."""
         self.permissions = [
             'profiles.create_user',
-            'profiles.can_edit_profiles']
+            'profiles.can_edit_profiles',
+            'profiles.can_change_mentor']
 
     def test_admin_group_has_all_permissions(self):
         """Test that admin group has all permissions."""
         admin = UserFactory.create(groups=['Admin'])
         for permission in self.permissions:
             ok_(admin.has_perm(permission))
 
-    def test_council_group_has_no_permissions(self):
-        """Test that council group has no permissions."""
+    def test_council_group_has_mentor_change_permissions(self):
+        """Test that council group has mentor change permissions."""
         councelor = UserFactory.create(groups=['Council'])
         for permission in self.permissions:
-            ok_(not councelor.has_perm(permission))
+            if permission == 'profiles.can_change_mentor':
```

Comment:
I would avoid having an `if` in a test. Probably this means that you can break this test into two and test both cases.